### ISSUE TITLE
Script

### ISSUE THREAD

**Issue by MasakrPetrov** [noxsicarius/Epoch-Admin-Tools#324]
Hi,

i try install a admintools and still i have a problem. I check everything :-(   

Thx for any help

 

script.log:

02.04.2018 09:47:41: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #1 "MagazineCargoGlobal [_itemOut,_countOut]};
case 5: {_item addBackpackCargoGlobal [_itemOut,_countOut]}; 
};
} else {
_item addMa"
02.04.2018 09:54:51: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #1 "MagazineCargoGlobal [_itemOut,_countOut]};
case 5: {_item addBackpackCargoGlobal [_itemOut,_countOut]}; 
};
} else {
_item addMa"
02.04.2018 10:08:18: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #1 "MagazineCargoGlobal [_itemOut,_countOut]};
case 5: {_item addBackpackCargoGlobal [_itemOut,_countOut]}; 
};
} else {
_item addMa"
02.04.2018 11:30:52: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #1 "MagazineCargoGlobal [_itemOut,_countOut]};
case 5: {_item addBackpackCargoGlobal [_itemOut,_countOut]}; 
};
} else {
_item addMa"
02.04.2018 12:49:35: Petr (86.49.247.48:49818) 7b7b7791b1e2fd639f8249296a235495 - #40 "nBuild = false;

DZE_buildMaxMoveDistance = 20;

EAT_AdminMenuHotkey = true; 
EAT_ActionMenuHotkey = true; 

EAT_HumanityG"
02.04.2018 12:54:46: Petr (86.49.247.48:49818) 7b7b7791b1e2fd639f8249296a235495 - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"

**Comment by noxsicarius** [comment]
Try the new test branch updates. There are brand new code changes in there.

**Comment by MasakrPetrov** [comment]
THX for answer, but...I use a new server install and i tried last test branch and everything is still the same :-(   **BattleEye: restriction 40**

**Comment by noxsicarius** [comment]
Did you try the new filters? They should be set up to fix that.

**Comment by MasakrPetrov** [comment]
I have reinstalled the server and i used a files and instruction from https://github.com/noxsicarius/Epoch-Admin-Tools/tree/TestBranch

its ok or.... ?

**Comment by noxsicarius** [comment]
Does that restriction happen when you load in? as long as you replaced your filters with the files from that download it should be working.

**Comment by MasakrPetrov** [comment]
Yes, the sign will be displayed when it is loaded. It will stop in the "authentication" step. Performs 4 attempts and "Failed to verify. Please Disconnect. Try later "

Again I copied the files from the link.

In the last attempt without "Restriction...."

THX  for answer :-(

**Comment by noxsicarius** [comment]
try providing more of the log. it seems like it may be missing the actual error cause unless they have modified the log prints in 106

**Comment by MasakrPetrov** [comment]
Hi. This is complete screen of script.log. Or another file ?

**SCRIPT.LOG**

07.04.2018 11:21:49: Petr (86.49.252.29:63024) 7b7b7791b1e2fd639f8249296a235495 - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
07.04.2018 19:32:48: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #1 "MagazineCargoGlobal [_itemOut,_countOut]};
case 5: {_item addBackpackCargoGlobal [_itemOut,_countOut]}; 
};
} else {
_item addMa"
12.04.2018 21:01:50: Petr (86.49.246.215:5876) 7b7b7791b1e2fd639f8249296a235495 - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
13.04.2018 00:10:56: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
13.04.2018 00:14:57: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
13.04.2018 14:49:28: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
13.04.2018 16:33:30: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
13.04.2018 16:41:43: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
13.04.2018 16:58:12: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
13.04.2018 21:40:47: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
14.04.2018 21:36:53: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
15.04.2018 13:21:59: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
15.04.2018 18:15:13: fark (109.202.34.0:2304) ceccaf766cb246db6020a0c72b64dadd - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
17.04.2018 18:56:53: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
18.04.2018 20:43:19: petr (93.118.27.108:27674) 189cd8db8a2b7551d7ff2c55fb6923fe - #40 "ate.sqf","admintools\antihack\antihack.sqf","admintools\actionMenu\ActionsMenu.sqf","admintools\actionMenu\contactAdmin.sqf","ad"
18.04.2018 21:27:38: Daiver (178.155.4.207:12942) 9080af68e0d3d1256711f672cac9a4e0 - #40 "nBuild = false;


DZE_buildMaxMoveDistance = 20;




EAT_AdminMenuHotkey = true; 
EAT_ActionMenuHotkey = true; 



EAT_HumanityG"
19.04.2018 15:27:02: Petr (86.49.247.165:25024) 7b7b7791b1e2fd639f8249296a235495 - #40 "nBuild = false;


DZE_buildMaxMoveDistance = 20;




EAT_AdminMenuHotkey = true; 
EAT_ActionMenuHotkey = true; 



EAT_HumanityG"

**Comment by MasakrPetrov** [comment]
Please, I still need a help with my problem...  :-/
